src/components/message-display-rules/NewRuleForm.tsx

```diff
@@ -0,0 +1,123 @@
+/** *****************************************************************************
+ * Copyright 2020-2020 Exactpro (Exactpro Systems Limited)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ ***************************************************************************** */
+
+import { nanoid } from 'nanoid';
+import React, { useRef, useState } from 'react';
+import { useMessageDisplayRulesStore } from '../../hooks';
+import { MessageDisplayRule, MessageViewType } from '../../models/EventMessage';
+import AutocompleteInput from '../util/AutocompleteInput';
+import Select from '../util/Select';
+
+type NewRuleFormProps = {
+	sessions: string[];
+	rule?: MessageDisplayRule | null;
+	stopEdit?: () => void;
+};
+
```

Comment:
this should be useRef<HTMLInputElement>(null)

add layout effect which will focus input on mount